worker/updates.js

```diff
@@ -48,35 +43,54 @@ const purgeFastly = async uuid => {
 
 	// GET once from fastly (but don't wait) to ensure the fastly shield has
 	// fresh content
-	fetch(purgeUrl(uuid));
+	fetch(purgeUrl);
 
-	await handleResponse(response);
-};
+	return handleResponse(response, purgeUrl);
+})());
 
-const purgeAmp = async uuid => {
-	const {hostname, pathname} = url.parse(purgeUrl(uuid));
+const purgeAmp = purgeUrl => logPurge({
+	purgeUrl,
+	name: 'AMP',
+})((async () => {
+	const {hostname, pathname} = url.parse(purgeUrl);
 	const response = fetch(`https://cdn.ampproject.org/update-ping/i/s/${hostname}${pathname}`);
 
-	await handleResponse(response);
-};
+	return handleResponse(response, purgeUrl);
+})());
 
-const purge = uuid => {
-	logPurge({
-		uuid,
-		name: 'Fastly',
-	})(purgeFastly(uuid));
+const formatPurgeUrl = (uuid, host) => url.format({
+	protocol: 'https',
+	host,
+	pathname: `/content/${uuid}`,
```

Comment:
so, this has always been silently broken, i.e. not purging the right thing from fastly _or_ amp. fml